app/src/main/java/com/andela/art/MainActivity.java

```diff
@@ -1,13 +1,23 @@
 package com.andela.art;
 
-import android.support.v7.app.AppCompatActivity;
+import android.content.Intent;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.andela.art.Login.LoginActivity;
+
+
+/**
+ * Created by asce1062 on 23/02/2018.
+ */
 
 public class MainActivity extends AppCompatActivity {
```

Comment:
This class seems to be doing nothing since LoginActivity is the first activity, it could be made the `launcher activity`